src/extractcode/extract.py

```diff
@@ -97,7 +97,21 @@
 ExtractEvent = namedtuple('ExtractEvent', 'source target done warnings errors')
 
 
-def extract(location, kinds=extractcode.default_kinds, recurse=False):
+def extract(location, kinds=extractcode.default_kinds, recurse=False, replace_originals=False):
```

Comment:
Do you mind  moving up the docstring that is now attached to the new `extract_files` function below https://github.com/nexB/scancode-toolkit/pull/1893/files#diff-16d915471be787f821dcd1d38891c2bdR116?

You could also add a doc note about the new `replace_originals` argument 
there.